swatch-producer-azure/src/main/java/com/redhat/swatch/azure/kafka/streams/BillableUsageAggregationStreamProperties.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.azure.kafka.streams;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import lombok.Data;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+@Data
+@ApplicationScoped
+public class BillableUsageAggregationStreamProperties {
```

Comment:
All the properties can use default visibility (drop `private` qualifier) [and intellij recommends this for quarkus].